.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thanks for submitting a pull request! Before pressing that tempting green submit
+button, please make sure you check off all the following boxes:
+
+- [ ] All existing unit tests pass in all supported Python versions (2.6, 2.7,
+      and 3.3+)
+- [ ] New tests have been added to verify the new functionality / bug fix
+- [ ] All code works on at least Windows, Mac OS X, and Linux
```

Comment:
I am not a fan of this check box. It will probably be hard for contributors to get their hands on a development environment for all three of these OS's. This is more of a check that we should pay attention to.